kolibri/core/assets/src/utils/syncTaskUtils.js

```diff
@@ -1,7 +1,45 @@
-import coreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
-import taskStrings from 'kolibri.coreVue.mixins.commonTaskStrings';
-import bytesForHumans from 'kolibri.utils.bytesForHumans';
-import { TaskStatuses, TaskTypes } from '../constants';
+import commonCoreStrings from '../mixins/commonCoreStrings';
+import taskStrings from '../mixins/taskStrings';
```

Comment:
I know it was like this when you got here, but for the taskStrings at least, the mixins module exposes an export for the function: https://github.com/learningequality/kolibri/blob/60dd14c37ecf8d3971b5013290c8a40ca9599668/kolibri/core/assets/src/mixins/taskStrings.js#L119

Which should avoid having to do an awkward destructing off the methods property of the mixin itself.

Would be good to do something similar for commonCoreStrings.

Reply:
The `coreString` signature maps directly to `$tr` a la `(key : string, args: object)` rather than taking some arbitrary number of nameless args - I'll apply the same here to make it more readable / consistent with the other code.

The `commonCoreStrings` pattern though for using it in JS is like:

```
import { coreStrings } from 'kolibri...commonScoreStrings'; // imports the created translator

coreStrings.$tr(...);
```

So maybe it'd be good to export a getter function there as well and unify the pattern altogether. In which case maybe I can move this to a separate issue re: strings things to do before release?

Reply:
Yeah, I think having the explicit args is better - might be good to open a follow up and hash it out there, yes. I can imagine there will probably be other things to consider before we start moving stuff around more!